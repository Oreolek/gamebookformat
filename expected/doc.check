examples/doc.json: Could not reach section 'custtut' from start.
examples/doc.json: Could not reach section 'basictut' from start.
examples/doc.json: Could not reach section 'dyntut' from start.
examples/doc.json: Could not reach section 'install' from start.
examples/doc.json: Could not reach section 'tutorials' from start.
examples/doc.json: Could not reach section 'advancedtut' from start.
examples/doc.json: Could not reach section 'reference' from start.
