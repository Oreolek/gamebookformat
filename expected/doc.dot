digraph gamebook {
Getting Started.Installation.Tutorials.3->4
3->5
3->6
3->7
3->8
Basic Tutorial.Advanced Tutorial.5->4
Dynamic Tutorial.Customization TutorialReference.}
