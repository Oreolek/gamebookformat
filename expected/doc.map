{"custtut": 7, "basictut": 4, "reference": 8, "start": 1, "dyntut": 6, "install": 2, "tutorials": 3, "advancedtut": 5}