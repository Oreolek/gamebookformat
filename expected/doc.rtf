{\rtf1\ansi\ansicpg1252\cocoartf1038\cocoasubrtf360
{\fonttbl\f0\fswiss\fcharset0 Helvetica;}
{\colortbl;\red255\green255\blue255;}
\paperw11900\paperh16840\margl1440\margr1440\vieww14140\viewh14860\viewkind0
\pard\tx566\tx1133\tx1700\tx2267\tx2834\tx3401\tx3968\tx4535\tx5102\tx5669\tx6236\tx6803\ql\qnatural\pardirnatural

\f0\b\fs24 \cf0

\b \qc gamebookformat 1.0 documentation
\b0\
\
\b \qc Introduction
\b0\



\ql  This is the documentation for gamebookformat 1.0. It was itself generated using gamebookformat (or more precisely the formatgamebook.py tool).  \
\
\b Turn to 1 to begin.
\b0\
\
\b \qc {\*\bkmkstart s1}1{\*\bkmkend s1}
\b0\
\ql  Getting Started.  \
\

\b \qc {\*\bkmkstart s2}2{\*\bkmkend s2}
\b0\
\ql  Installation.  \
\

\b \qc {\*\bkmkstart s3}3{\*\bkmkend s3}
\b0\
\ql  Tutorials. There are 4 tutorials of increasing complexity currently included in this document: \b {\field{\*\fldinst  REF s4 \\h }{\fldrslt The Basic Tutorial}}
\b0

 shows how to make a simple static gamebook for printing or reading on a screen. \b {\field{\*\fldinst  REF s5 \\h }{\fldrslt The Advanced Tutorial}}
\b0

 adds many formatting tricks to make more interesting gamebooks, but still only very static. \b {\field{\*\fldinst  REF s6 \\h }{\fldrslt The Dynamic Tutorial}}
\b0

 finally explains how to add mark-up to your gamebooks to be make dynamic HTML gamebooks that can be played in a browser (while still work well if it is printed on paper or viewed in a simple ebook reader). Finally \b {\field{\*\fldinst  REF s7 \\h }{\fldrslt The Customization Tutorial}}
\b0

 shows how to override the default templates to make small or big changes to how gamebooks are rendered. The tutorials do not cover all features of gamebookformat, so also have a look at the included examples and the \b {\field{\*\fldinst  REF s8 \\h }{\fldrslt Reference section below}}
\b0

 to learn about all the things the tools can do. \
\

\b \qc {\*\bkmkstart s4}4{\*\bkmkend s4}
\b0\
\ql  Basic Tutorial. This tutorial will guide you through creating a simple gamebook with linked sections of text with some simple formatting, resulting in static html and rtf documents that can be easily navigated for manual play printed on paper or on any computer or ebook reader. \
\

\b \qc {\*\bkmkstart s5}5{\*\bkmkend s5}
\b0\
\ql  Advanced Tutorial. This tutorial continues the \b {\field{\*\fldinst  REF s4 \\h }{\fldrslt Basic Tutorial}}
\b0

, only adding some more details to do more advanced formatting like adding images to books or how to make links that display text instead of numbers. \
\

\b \qc {\*\bkmkstart s6}6{\*\bkmkend s6}
\b0\
\ql  Dynamic Tutorial.  \
\

\b \qc {\*\bkmkstart s7}7{\*\bkmkend s7}
\b0\
\ql  Customization Tutorial. \
\

\b \qc {\*\bkmkstart s8}8{\*\bkmkend s8}
\b0\
\ql  Reference. \
\

}
