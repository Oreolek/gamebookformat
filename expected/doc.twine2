<tw-storydata name="gamebookformat 1.0 documentation" startnode="1" creator="Twine" creator-version="2.0" format="Harlowe" options="">
<style role="stylesheet" id="twine-user-stylesheet" type="text/twine-css"></style><script role="script" id="twine-user-script" type="text/twine-javascript"></script>
<tw-passagedata pid="1" name="1" tags="" position="10,10"> <b>Getting Started.</b> </tw-passagedata>
<tw-passagedata pid="2" name="2" tags="" position="20,20"> <b>Installation.</b> </tw-passagedata>
<tw-passagedata pid="3" name="3" tags="" position="30,30"> <b>Tutorials.</b> There are 4 tutorials of increasing complexity currently included in this document: [[The Basic Tutorial->4]]
 shows how to make a simple static gamebook for printing or reading on a screen. [[The Advanced Tutorial->5]]
 adds many formatting tricks to make more interesting gamebooks, but still only very static. [[The Dynamic Tutorial->6]]
 finally explains how to add mark-up to your gamebooks to be make dynamic HTML gamebooks that can be played in a browser (while still work well if it is printed on paper or viewed in a simple ebook reader). Finally [[The Customization Tutorial->7]]
 shows how to override the default templates to make small or big changes to how gamebooks are rendered. The tutorials do not cover all features of gamebookformat, so also have a look at the included examples and the [[Reference section below->8]]
 to learn about all the things the tools can do.</tw-passagedata>
<tw-passagedata pid="4" name="4" tags="" position="40,40"> <b>Basic Tutorial.</b> This tutorial will guide you through creating a simple gamebook with linked sections of text with some simple formatting, resulting in static html and rtf documents that can be easily navigated for manual play printed on paper or on any computer or ebook reader.</tw-passagedata>
<tw-passagedata pid="5" name="5" tags="" position="50,50"> <b>Advanced Tutorial.</b> This tutorial continues the [[Basic Tutorial->4]]
, only adding some more details to do more advanced formatting like adding images to books or how to make links that display text instead of numbers.</tw-passagedata>
<tw-passagedata pid="6" name="6" tags="" position="60,60"> <b>Dynamic Tutorial.</b> </tw-passagedata>
<tw-passagedata pid="7" name="7" tags="" position="70,70"> <b>Customization Tutorial</b>.</tw-passagedata>
<tw-passagedata pid="8" name="8" tags="" position="80,80"> <b>Reference.</b></tw-passagedata>
</tw-storydata>
