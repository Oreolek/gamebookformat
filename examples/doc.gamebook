title = gamebookformat 1.0 documentation
author = Pelle Nilsson

= Introduction
This is the documentation for gamebookformat 1.0. It was itself generated using
gamebookformat (or more precisely the formatgamebook.py tool).

* 1 start
[b]Getting Started.[b]

* 2 install
[b]Installation.[b]

* 3 tutorials
[b]Tutorials.[/b]
There are 4 tutorials of increasing complexity currently included
in this document: [[basictut][The Basic Tutorial]] shows how to make a simple
static gamebook for printing or reading on a screen.
[[advancedtut][The Advanced Tutorial]] adds many formatting tricks to make more
interesting gamebooks, but still only very static.
[[dyntut][The Dynamic Tutorial]] finally explains how to add mark-up to your
gamebooks to be make dynamic HTML gamebooks that can be played
in a browser (while still work well if it is printed on paper or
viewed in a simple ebook reader). Finally
[[custtut][The Customization Tutorial]] shows how to override the default templates
to make small or big changes to how gamebooks are rendered. The
tutorials do not cover all features of gamebookformat, so also
have a look at the included examples and the [[reference][Reference section below]]
to learn about all the things the tools can do.
* 4 basictut
[b]Basic Tutorial.[/b]
This tutorial will guide you through creating a simple gamebook with linked sections of text
with some simple formatting,
resulting in static html and rtf documents that can be easily navigated for manual play
printed on paper or on any computer or ebook reader.
* 5 advancedtut
[b]Advanced Tutorial.[/b]
This tutorial continues the [[basictut][Basic Tutorial]], only adding some more
details to do more advanced formatting like adding images to
books or how to make links that display text instead of numbers.
* 6 dyntut
[b]Dynamic Tutorial.[/b]

* 7 custtut
[b]Customization Tutorial[/b]. 
* 8 reference
[b]Reference.[/b]
