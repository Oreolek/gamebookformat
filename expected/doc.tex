\documentclass[a5paper,onecolumn]{book}

\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage[hidelinks]{hyperref}
\usepackage{graphicx}

\usepackage[top=3.3cm, bottom=3.3cm, left=2cm, right=2cm]{geometry}
\newif\ifpdf
\ifx\pdfoutput\undefined
  \pdffalse
\else
  \ifnum\pdfoutput=1
    \pdftrue
  \else
    \pdffalse
  \fi
\fi

\title{gamebookformat 1.0 documentation}
\author{Pelle Nilsson}
\date{}

\newcounter{sectionnr}

\begin{document}

\maketitle

\clearpage

\thispagestyle{empty}

\pagestyle{empty}
\subsection*{\begin{center} \textbf{Introduction} \end{center}}


 \noindent
 This is the documentation for gamebookformat 1.0. It was itself generated using gamebookformat (or more precisely the formatgamebook.py tool). 
\vspace{1em}

Turn to 1 to begin.
\phantomsection
\refstepcounter{sectionnr}
\label{section1}
\subsection*{\begin{center} \textbf{1} \end{center}}

 \noindent
 Getting Started. 
\vspace{1em}
\phantomsection
\refstepcounter{sectionnr}
\label{section2}
\subsection*{\begin{center} \textbf{2} \end{center}}

 \noindent
 Installation. 
\vspace{1em}
\phantomsection
\refstepcounter{sectionnr}
\label{section3}
\subsection*{\begin{center} \textbf{3} \end{center}}

 \noindent
 Tutorials. There are 4 tutorials of increasing complexity currently included in this document: The Basic Tutorial (\textbf{\autoref{section4}})
 shows how to make a simple static gamebook for printing or reading on a screen. The Advanced Tutorial (\textbf{\autoref{section5}})
 adds many formatting tricks to make more interesting gamebooks, but still only very static. The Dynamic Tutorial (\textbf{\autoref{section6}})
 finally explains how to add mark-up to your gamebooks to be make dynamic HTML gamebooks that can be played in a browser (while still work well if it is printed on paper or viewed in a simple ebook reader). Finally The Customization Tutorial (\textbf{\autoref{section7}})
 shows how to override the default templates to make small or big changes to how gamebooks are rendered. The tutorials do not cover all features of gamebookformat, so also have a look at the included examples and the Reference section below (\textbf{\autoref{section8}})
 to learn about all the things the tools can do.
\vspace{1em}
\phantomsection
\refstepcounter{sectionnr}
\label{section4}
\subsection*{\begin{center} \textbf{4} \end{center}}

 \noindent
 Basic Tutorial. This tutorial will guide you through creating a simple gamebook with linked sections of text with some simple formatting, resulting in static html and rtf documents that can be easily navigated for manual play printed on paper or on any computer or ebook reader.
\vspace{1em}
\phantomsection
\refstepcounter{sectionnr}
\label{section5}
\subsection*{\begin{center} \textbf{5} \end{center}}

 \noindent
 Advanced Tutorial. This tutorial continues the Basic Tutorial (\textbf{\autoref{section4}})
, only adding some more details to do more advanced formatting like adding images to books or how to make links that display text instead of numbers.
\vspace{1em}
\phantomsection
\refstepcounter{sectionnr}
\label{section6}
\subsection*{\begin{center} \textbf{6} \end{center}}

 \noindent
 Dynamic Tutorial. 
\vspace{1em}
\phantomsection
\refstepcounter{sectionnr}
\label{section7}
\subsection*{\begin{center} \textbf{7} \end{center}}

 \noindent
 Customization Tutorial.
\vspace{1em}
\phantomsection
\refstepcounter{sectionnr}
\label{section8}
\subsection*{\begin{center} \textbf{8} \end{center}}

 \noindent
 Reference.
\vspace{1em}
\end{document}
